File: 2578. Split With Minimum Sum/solution.js
/**
 * @param {number} num
 * @return {number}
 */
var splitNum = function(num) {
  let stringNum = num.toString();
  let sortedNum = stringNum.split('').sort().join('');
  
  let num1 = "";
  let num2 = "";
  for (let index = 0; index < sortedNum.length; index++){
    let numValue = sortedNum[index];
    if (index % 2 == 0){
      num1 += numValue;
    }
    else{
      num2 += numValue;
    }
  }
      
  return parseInt(num1) + parseInt(num2);
};

File: 2578. Split With Minimum Sum/solution.cpp
class Solution {
public:
  int splitNum(int num) {
    string stringNum = to_string(num);
    sort(stringNum.begin(), stringNum.end());
    
    string num1 = "";
    string num2 = "";
    for (int index = 0; index < stringNum.size(); index++){
      char numValue = stringNum[index];
      if (index % 2 == 0){
        num1 += numValue;
      }
      else{
        num2 += numValue;
      }
    }
        
    return stoi(num1) + stoi(num2);
  }
};